example/centos/common/partition-table/x86_64/minimal-raw.yaml

```diff
@@ -0,0 +1,41 @@
+otk.define:
+  filesystem:
+    otk.external.otk-gen-partition-table:
+      modifications:
+        ${filesystem.modifications}
+      properties:
+        type: gpt
+        bios: false
```

Comment:
It seems the only difference between the partition table of x86/arm64:
```yaml
--- x86_64/default.yaml	2024-09-30 09:37:17.920536402 +0200
+++ aarch64/default.yaml	2024-09-30 09:37:17.920536402 +0200
@@ -9,7 +9,7 @@
         default_size: "10 GiB"
         uuid: D209C89E-EA5E-4FBD-B161-B461CCE297E0
         create:
-          bios_boot_partition: true
+          bios_boot_partition: false
           esp_partition: true
           esp_partition_size: "200 MiB"
       partitions:
```
(same for default and vhd) but given that we already duplicate this for the other I guess it's okay.

Reply:
This will definitely be part of the deduplication stuff :)